### ISSUE TITLE
Lua error.

### ISSUE THREAD

**Issue by Andols0** [Xam-insight/DeadPoolGambling#1]
Installed the addon today and getting this error from time to time.
I don't really do anything to trigger it, it just appears.

```
47x Deadpool_CustomAchiever/Sharing.lua:11: attempt to concatenate local 'info' (a nil value)
[string "@Deadpool_CustomAchiever/Sharing.lua"]:11: in function <Deadpool_CustomAchiever/Sharing.lua:10>
[string "@Deadpool_CustomAchiever/Sharing.lua"]:15: in function `CustAc_SendCallForUsers'
[string "@Deadpool_CustomAchiever/CustomAchiever.lua"]:51: in function `?'
[string "@Auctioneer/Libs/LibDataBroker/CallbackHandler/CallbackHandler-1.0-8.lua"]:109: in function <...ibDataBroker/CallbackHandler/CallbackHandler-1.0.lua:109>
[string "=[C]"]: ?
[string "@Auctioneer/Libs/LibDataBroker/CallbackHandler/CallbackHandler-1.0-8.lua"]:19: in function <...ibDataBroker/CallbackHandler/CallbackHandler-1.0.lua:15>
[string "@Auctioneer/Libs/LibDataBroker/CallbackHandler/CallbackHandler-1.0-8.lua"]:54: in function `Fire'
[string "@Bazooka/libs/Ace3/AceEvent-3.0-4/AceEvent-3.0.lua"]:120: in function <...dOns/Bazooka/libs/Ace3/AceEvent-3.0/AceEvent-3.0.lua:119>

Locals:
info = nil
aTarget = nil
messageType = "CallUsers"
```

**Comment by Xam-insight** [comment]
Thank you for the feedback.
I’ll check it out.

**Comment by Xam-insight** [comment]
Fixed in v1.10.7.

**Comment by Xam-insight** [comment]
Fixed in v1.10.7.
